src/TrackPropDlg.cpp

```diff
@@ -1734,17 +1730,9 @@ void TrackPropDlg::OnOKBtnClick(wxCommandEvent& event) {
   //    Look in the track list to be sure the track is still available
   //    (May have been deleted by RouteManagerDialog...)
 
-  bool b_found_track = false;
-  wxTrackListNode* node = pTrackList->GetFirst();
-  while (node) {
-    Track* ptrack = node->GetData();
-
-    if (ptrack == m_pTrack) {
-      b_found_track = true;
-      break;
-    }
-    node = node->GetNext();
-  }
+  bool b_found_track =
+    std::find(g_TrackList.begin(), g_TrackList.end(), m_pTrack) ==
+    g_TrackList.end();
```

Comment:
hm... b_found_track is true if std::find() == g_TrackList.end()... doesn't seem sound.  Or am I missing something?

Reply:
Thank you, fixed.